sample/src/androidTest/kotlin/com/agoda/sample/ConfiguratorTestActivityTest.kt

```diff
@@ -0,0 +1,124 @@
+package com.agoda.sample
+
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import android.util.Log
+import com.agoda.kakao.configurator.KakaoConfigurator
+import com.agoda.kakao.screen.Screen
+import com.agoda.sample.screen.ConfiguratorTestActivityScreen
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ConfiguratorTestActivityTest {
+
+    init {
+        KakaoConfigurator.clearConfigure {
```

Comment:
I think that @Unlimity wanna have smth like 

```
Kakao {
  onViewInteraction {
  }

...
}
```

isn't it?

Reply:
My concept is so similar =)